tools/json2cbor/json2cbor.c

```diff
@@ -448,8 +448,10 @@ int main(int argc, char **argv)
         buffer = NULL;
         do {    // it the hard way
             buffer = realloc(buffer, buffersize + chunk);
-            if (buffer == NULL)
+            if (buffer == NULL) {
                 perror("malloc");
```

Comment:
Not sure giving perror a string is valuable (since the message becomes "malloc:Out of memory", but this is existing.